workflow/controller/operator_test.go

```diff
@@ -4235,24 +4223,24 @@ spec:
 	// Parent dag node has no pod
 	parentNode, err := woc.wf.GetNodeByName("dag-events")
 	if err != nil {
-		assert.NoError(t, err)
+		require.NoError(t, err)
 	}
```

Comment:
```suggestion
	require.NoError(t, err)
```

to apply same as above